packages/autocomplete-plugin-algolia-insights/src/types/AlgoliaInsightsHit.ts

```diff
@@ -2,4 +2,8 @@ export type AlgoliaInsightsHit = {
   objectID: string;
   __autocomplete_indexName: string;
   __autocomplete_queryID: string;
+  __autocomplete_algoliaResultsMetadata: {
```

Comment:
```suggestion
  __autocomplete_algoliaCredentials: {
```

this could also work I think, or otherwise two separate keys